src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -67,9 +67,9 @@ public PublicationController(PublicationService publicationService) {
         @RequestHeader(PublicationConfiguration.TYPE_HEADER) ArtefactType type,
         @RequestHeader(value = PublicationConfiguration.SENSITIVITY_HEADER, required = false) Sensitivity sensitivity,
         @RequestHeader(value = PublicationConfiguration.LANGUAGE_HEADER, required = false) Language language,
-        @RequestHeader(value = PublicationConfiguration.DISPLAY_FROM_HEADER, required = false)
+        @RequestHeader(PublicationConfiguration.DISPLAY_FROM_HEADER)
```

Comment:
Looks good! I think we need to add some integration tests for these :)

Reply:
Done